app/models/metric/rollup.rb

```diff
@@ -152,6 +155,14 @@ def self.rollup_hourly(obj, hour, _interval_name, _time_profile, new_perf, orig_
     new_perf.reverse_merge!(orig_perf)
     new_perf.merge!(Metric::Processing.process_derived_columns(obj, new_perf, hour)) unless DERIVED_COLS_EXCLUDED_CLASSES.include?(obj.class.base_class.name)
 
+    # Add metric data collected from vmdb
+    if orig_perf[:resource_type] == "ContainerNode"
```

Comment:
@yaacov we want to count how many new `container_groups` were created per-provider, so it should be attached to the EMS.